src/EnergyPlus/DataHVACGlobals.hh

```diff
@@ -372,52 +353,68 @@ namespace HVAC {
     Real64 constexpr MaxCoolVolFlowPerRatedTotCap2(0.00004026);  // m3/s per watt = 300 cfm/ton
     Real64 constexpr MinOperVolFlowPerRatedTotCap2(0.00001342);  // m3/s per watt = 100 cfm/ton
 
-    // dx coil type (DXCT)
-    int constexpr RegularDXCoil(1); // Regular DX coils or mixed air dx coils
-    int constexpr DOASDXCoil(2);    // 100% DOAS DX coils
-
-    // Parameters describing Heat Exchanger types
-    int constexpr NumHXTypes(3);
+    constexpr std::array<Real64, 2> MaxRatedVolFlowPerRatedTotCap = {MaxRatedVolFlowPerRatedTotCap1, MaxRatedVolFlowPerRatedTotCap2};
```

Comment:
These arrays of `constexpr Real64` can themselves be `constexpr std::array`s and don't need to be made into state variables.